packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -50,28 +50,32 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     // Reserves that are not yet utilized but are pre-allocated for a pending relay.
     uint256 public pendingReserves;
 
+    // If this pool contains WETH. If the withdrawn token is WETH then unwrap and send ETH  when finalizing withdrawal.
+    bool public isWethPool;
```

Comment:
Moved these variables to save a few thousand gas since they're often read together.